src/WinRT.Runtime/Projections/IReferenceArray.net5.cs

```diff
@@ -83,9 +145,18 @@ public static object FromAbi(IntPtr ptr)
 
         public static unsafe object GetValue(IInspectable inspectable)
         {
+#if NET

```

Comment:
I need to double check what our behavior here is today when boxed arrays are marshaled from native side.

Reply:
This type should never be used on AOT (I left some comments above the type explaining it). Unless I'm missing some way it could still somehow end up being referenced on NAOT? I couldn't find any possible entry points though 🤔